File: DemoServer/AdditionalLanguages/python/Basics/loadAndEditDocument/loadAndEditDocument.py
from demo_example import Example, RunParamsBase
from models import Company

class RunParams(RunParamsBase):
    def __init__(self, company_name: str):
        self.company_name = company_name

class LoadAndEditDocument(Example):
    def run(self, run_params: RunParams) -> None:
        company_name = run_params.company_name

        #region Demo
        with self.document_store_holder.store().open_session() as session:
            #region Step_1
            company = session.load("companies/5-A", Company)
            #endregion

            #region Step_2
            company.Name = company_name
            #endregion

            #region Step_3
            session.save_changes()
            #endregion
        #endregion

File: DemoServer/AdditionalLanguages/php/basics/loadAndEditDocument/LoadAndEditDocument.php
<?php

namespace RavenDB\Demo\basics\loadAndEditDocument;

//region Usings
use RavenDB\Documents\Session\DocumentSessionInterface;
//endregion
Use RavenDB\Demo\common\DocumentStoreHolder;
use RavenDB\Demo\common\models\Company;

class LoadAndEditDocument
{
    public function __invoke(RunParams $runParams)
    {
        $companyName = $runParams->getCompanyName();

        //region Demo
        $session = DocumentStoreHolder::getStore()->openSession();
        try {
            //region Step_1
            $company = $session->load(Company::class, "companies/5-A");
            //endregion

            //region Step_2
            $company->setName($companyName);
            //endregion

            //region Step_3
            $session->saveChanges();
            //endregion
            
        } finally {
            $session->close();
        }
        //endregion
    }
}
